Fix the place of the banner survey

monikappv opened the pull request:
# Overview

* I noticed that the banner is on the wrong page + also shown in a wrong way.. apparently I have somehow put the html code in the wrong file.. this PR should fix this problem and put it on the home page in a right way.

---

Please preserve this line to notify @lwinfree (lead of this repository)

lwinfree commented:
hi @monikappv I just noticed this PR as I was about to open a related issue. The survey banner isn't rendering correctly on the reproducible research page or on the field guide. However, it seems OK when I view it on a smaller screen. Is this PR to fix that? Or is this a different problem? Let me know if I can help - thanks!

monikappv commented:
hey @lwinfree ..at the beginning I had added the banner to the wrong page (field guide), after that I moved it to the home page and sent the pull request... I am a little bit confused re this situation, since I actually changed only the home page script (to add the banner) but the banner is appearing on all of the pages. Anyway... the purpose of the PR was to fix that - yes. And I hope it does.

lwinfree commented:
this looks good to me @monikappv! Go ahead and merge (unless you are waiting on something - but it looks good to me)
